src/CMakeLists.txt

```diff
@@ -7,8 +7,14 @@ project (libproject)
 set (SOURCES
 	libproject.cpp)
 
+#setting visibility of headers in project tree
+set (HEADERS
```

Comment:
**#bug**
Use file (GLOB)
Don't use relative paths (if you need some path configured it should be a variable)